### PR TITLE
Intro and personal info

### PR DESCRIPTION
**Pull request by guillermo-carrasco**

### CODE REVIEW

File: parts/intro.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
 # Introduction
 
 ## 1. What makes Python uniquely capable of handling NGS data?
-*Even to people outside science, I think this could be an interesting topic.*
+
+Python is a very versatile and easy to use language. It is famous for having a very
+quick learning curve:
+
+![Python XKCD](/assets/python.png?raw=True)
```
Comment (guillermo-carrasco) [inline parts/intro.md]: I think we should put this XKCD in a slide :stuck_out_tongue: , if its readable of course

Comment (robinandeer) [inline parts/intro.md]: Maybe people will just instantly recognize it!

File: parts/intro.md [hunk 2]
```diff
@@ -20,7 +32,22 @@ I believe this story resonates with many "scientific" programmers. Some of the m
 
 ### Guillermo Carrasco
 
-> How did Guillermo end up in life science?
+My name is Guillermo Carrasco and I am a software engineer infiltrated between biologists :)
```
Comment (robinandeer) [inline parts/intro.md]: Love this line :smile:

**Comment by robinandeer** [comment]
Maybe I'm biased but I think these intros are kind of sweet so it makes sense to work them into the talk; researchers turned "engineers", Python has monopoly in science etc.

Does it make any difference in what order our intros come? Otherwise the intuitive thing is to let you go first and I follow.

**Comment by robinandeer** [comment]
Time to fork this bad boy before it goes stale!

**Pull request merged**